### PR TITLE
Msgpack

### PR DESCRIPTION
**Pull request by wasbazi**
Implements the msgpack functionality as a reporter and as an input for the transformer.

### CODE REVIEW

File: lib/reporters/msgpackReporter.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var msgpack = require('msgpack')
+
+module.exports = {
+
+	reportResults: function(results, config, outputStream, callback) {
+		outputStream.write(msgpack.pack(JSON.stringify(results)));
```
Comment (jkrukoff) [inline lib/reporters/msgpackReporter.js]: Is this first serializing as JSON, then serializing the JSON as msgpack?

Comment (wasbazi) [inline lib/reporters/msgpackReporter.js]: Ignore that. *Waves hands* I'm fixing this in another commit.

**Comment by jkrukoff** [comment]
Looks good to me. Ideally, would like to get this merged by EOD, so can clear up the dashboard error.

**Comment by apriendeau** [comment]
:+1:

**Pull request merged**